style.css

```diff
@@ -1448,3 +1496,1226 @@ section */
 .btn-contact-text:hover {
   color: #ff6b00;
 }
+
+[class*="navD-"] {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [optional] Please, I noticed that at exactly this media screen there is an `horizontal overflow`. Sometimes the interaction isn’t obvious, and that can be a real issue when it comes to a horizontal scrolling. Users need to know and understand how to interact with your website to ensure a positive experience. ℹ️